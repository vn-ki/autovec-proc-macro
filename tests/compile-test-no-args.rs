use autovec::autovec;
#[autovec]
fn add() -> String {
    "hello world".to_string()
}

fn main() {
}
