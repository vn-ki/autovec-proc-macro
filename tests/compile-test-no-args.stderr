error: custom attribute panicked
 --> $DIR/compile-test-no-args.rs:2:1
  |
2 | #[autovec]
  | ^^^^^^^^^^
  |
  = help: message: expected at least one argument
